remotespark/pysparkkernel/kernel.js

```diff
@@ -0,0 +1 @@
+IPython.CodeCell.config_defaults.highlight_modes['magic_text/x-sql'] = {'reg':[/^%%sql/, /^%%hive/]};
```

Comment:
Cool! Did you want to change it to Hive? We'll get to that later (no one's asked for it yet), but nice to know it exists!